tutorial/testing.py

```diff
@@ -43,6 +44,39 @@
     can check the details from both [Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Headless_mode)
     and [Chrome](https://developers.google.com/web/updates/2017/04/headless-chrome).
 
+    Remote WebDriver is added in Dash *1.3.0*. You may have two use cases:
+
+    1. Run `pytest --remote --log-cli-level DEBUG -k bsly001` to grab a Chrome
+    WebDriver from a local hosted grid at `http://localhost:4444/wd/hub`
+    2. Run `pytest --webdriver Firefox --log-cli-level DEBUG --remote-url https://grid_provioder_endpoints`
```

Comment:
```suggestion
    2. Run `pytest --webdriver Firefox --log-cli-level DEBUG --remote-url https://grid_provider_endpoint`
```

Is `--log-cli-level` necessary here? Or is it just important for debugging connection to the remote? If the latter, perhaps move that to the Caveats section?

Similarly, I guess `--webdriver Firefox` is required if your grid is running FF? Perhaps just mention this then **"to connect with a remote grid in the cloud running Firefox"**

Reply:
yes, I think users would need the DEBUG level to see the communications on both sides. --webdriver here is to emphasize that you still need to set the browser type here, should also add a note that chrome is the default one